storage_tiering.cpp

```diff
@@ -856,6 +856,15 @@ namespace irods {
             // do not queue movement if data is on minimum tier or lower
             // TODO:: query for already queued movement?
             if (source_resource_tier <= low_tier_resource_tier) {
+                rodsLog(LOG_DEBUG,
+                        fmt::format("Migrating object [{}] from resource [{}] (tier [{}]) to resource [{}] (tier [{}]) "
+                                    "would move the object to a higher tier. Skipping restage.",
```

Comment:
Actually, this message isn't exactly accurate. The more common case here would be for the replica to already exist on the minimum restage tier, and that isn't portrayed by this message. I'll try to work it in.

Reply:
Done